packages/eth-json-rpc-provider/package.json

```diff
@@ -56,7 +56,7 @@
     "@metamask/json-rpc-engine": "^10.1.1",
     "@metamask/rpc-errors": "^7.0.2",
     "@metamask/utils": "^11.8.1",
-    "uuid": "^8.3.2"
+    "nanoid": "^3.3.8"
```

Comment:
Admittedly when I added `request` to the provider I swapped out `nanoid` for `uuid`. My thinking was that `uuid` was more standard. I'm just curious, what are your thoughts on this? (I can be convinced the other way)